core/services/ocr2/plugins/ocr2keeper/evm21/streams_lookup.go

```diff
@@ -529,15 +553,6 @@ func (r *EvmRegistry) multiFeedsRequest(ctx context.Context, ch chan<- MercuryDa
 				state = encoding.MercuryUnmarshalError
 				return err1
 			}
-			// in v0.3, if some feeds are not available, the server will only return available feeds, but we need to make sure ALL feeds are retrieved before calling user contract
-			// hence, retry in this case. retry will help when we send a very new timestamp and reports are not yet generated
-			if len(response.Reports) != len(sl.Feeds) {
```

Comment:
is it safe to say len(response.Reports) == len(sl.Feeds) since any feed will get a response, even tho it is `dataNotFound` in some cases?